src/server/config.js

```diff
@@ -1,15 +1,28 @@
-import { FlatFile, Mongo } from 'boardgame.io/server';
+import { FlatFile } from 'boardgame.io/server';
+
+class ModelFlatFile extends FlatFile {
+  async fetch(matchID, opts) {
+    var result = await super.fetch(matchID, opts);
+    if(opts.model) {
+      const key = ModelKey(matchID);
+      result.model = (await this.getItem(key));
+    }
+    return result
+  }
+  async setModel(id, model) {
+    const key = ModelKey(id);
+    return await this.setItem(key, model);
+  }
+}
+
+function ModelKey(matchID) {
```

Comment:
Convention is to start function names with a lower case letter. Maybe `getModelKey`?   And this function would be a member of `ModelFlatFile`, right?

Reply:
I just based it off what was in `cjs/server.js` which I think was done a bit weirdly because it's compiled typescript.

I think having getModelKey is more javascripty so I've changed it